magner/lib/controllers/store/store.ts

```diff
@@ -16,7 +16,20 @@ interface State {
   user: unknown | null,
   role: string | null,
 
+  /**
+   * used only for mobile screens. If true - sidebar opened
+   */
+  sidebarOpened: boolean,
   sidebarCollapsed: boolean,
+  /**
+   * used only for mobile screens. True if filters are hidden
+   */
+  filtersCollapsed: boolean,
+  /**
+   * if true - "collapse filters" button is shown in header
+   */
+  hasFilters: boolean;
+
```

Comment:
Suggestions: 
1. Move these settings to the (main) layout configuration. The store is not the place for this configuration.
2. `filtersCollapsed` is needed only in the Table pages. Therefore, move it into the table configs.
3. Rename mobile-only flags to `mobileSidebarOpened` and `mobileFiltersCollapsed`
4. `hasFilters` is a computable property, no need for the special flag